script.js

```diff
@@ -0,0 +1,79 @@
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+function getBooks() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  } return books;
+}
+
+function addBook(book) {
+  const books = getBooks();
+  books.push(book);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function removeBook(author) {
+  const books = getBooks();
+
+  books.forEach((book, index) => {
+    if (book.author === author) {
+      books.splice(index, 1);
+    }
+  });
+  localStorage.setItem('books', JSON.stringify(books));
+}
```

Comment:
[OPTIONAL]

- [ ] Please favor the use of array methods that doesn't mutate the array, e.g. `concat, filter, slice` instead of `push, pop, splice` and so on. 
- [ ] I suggest that you use unique `id` property for each book since it would make it easier to remove books with duplicate values.

See below example of both suggestions:
```javascript
const books = [];

const removeBook = (bookID) => {
  
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```